.prettierrc

```diff
@@ -25,6 +25,12 @@
       "options": {
         "printWidth": 200
       }
+    },
+    {
+      "files": ["devcontainer.json"],
+      "options": {
+        "trailingComma": "none"
+      }
```

Comment:
why do this though? Trailing commas make it easier to view diffs.

Reply:
yes, JSON doesn't have trailing comma support. also the CI automated the fix this way